tasks/42_first-n-smallest/first-n-smallest.js

```diff
@@ -0,0 +1,6 @@
+export function firstNSmallest(arr, num) {
+  const sortedArr = [...arr].sort();
+  return arr.filter((item) => {
+    return sortedArr.slice(0, num).includes(item);
```

Comment:
You're creating a new array on each iteration. This is not optimal. Consider creating this array out of iteration